ci/test.yml

```diff
@@ -25,6 +25,8 @@
     - sleep 5
     - docker-compose run test-run
   after_script:
+    - docker cp ${CI_JOB_ID}_trezor-user-env-unix_1:/trezor-user-env/logs/debugging.log .
+    - docker cp ${CI_JOB_ID}_trezor-user-env-unix_1:/trezor-user-env/docker/version.txt .
```

Comment:
Let's use some nice names. E.g.
```
    - docker cp ${CI_JOB_ID}_trezor-user-env-unix_1:/trezor-user-env/logs/debugging.log trezor-user-env-debugging.log
    - docker cp ${CI_JOB_ID}_trezor-user-env-unix_1:/trezor-user-env/docker/version.txt trezor-user-env-version.txt
```
I will merge then, thx!

Reply:
Force pushed the change as it's only renaming. should be good to go